src/components/AppHeader.js

```diff
@@ -1,38 +1,49 @@
-import React from "react"
-import { Link } from "react-router-dom"
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
 
 const AppHeader = () => {
+
+  const [activeLink, setActiveLink] = useState(0)
+  const links = [
+    {
+      display: "Home",
+      to: "/",
+    },
+    {
+      display: "About",
+      to: "/about",
+    },
+    {
+      display: "Portfolio",
+      to: "",
+    },
+    {
+      display: "Contact",
+      to: "/contact",
```

Comment:
do you think `label` better than `display`

Reply:
label is better

Reply:
fixed